doc/manual/writing_a_driver.rst

```diff
@@ -106,6 +113,34 @@ The controller's code would contain something similar to this: ::
         args = _get_args()
         simple_server_loop(Hello(), args.bind, args.port)
 
+The driver's server should be saved in ~/artiq-dev/artiq/artiq/devices/driver_name/server.py.
+
+__init__.py
+-----------
+This file must exist for porper integration with ARTIQ. It can be empty.
+~/artiq-dev/artiq/artiq/devices/driver_name/__init__.py
+
+Device Manager Udev Rules
```

Comment:
If we all agree on including a template for driver development at artiq/devices/example_artiq_device please add an example serial interface using serial_for_url().